libraries/AP_WindVane/AP_WindVane_config.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <AP_HAL/AP_HAL_Boards.h>
+
+#ifndef AP_WINDVANE_ENABLED
+#define AP_WINDVANE_ENABLED 1
+#endif
+
+#ifndef AP_WINDVANE_SITL_ENABLED
+#define AP_WINDVANE_SITL_ENABLED AP_WINDVANE_ENABLED
+#endif
+
+#ifndef AP_WINDVANE_RPM_ENABLED
+#define AP_WINDVANE_RPM_ENABLED AP_WINDVANE_ENABLED
+#endif
+
+#ifndef AP_WINDVANE_NMEA_ENABLED
+#define AP_WINDVANE_NMEA_ENABLED AP_WINDVANE_ENABLED
+#endif
+
+#ifndef AP_WINDVANE_HOME_ENABLED
+#define AP_WINDVANE_HOME_ENABLED AP_WINDVANE_ENABLED
+#endif
+
+#ifndef AP_WINDVANE_MODERNDEVICE_ENABLED
+#define AP_WINDVANE_MODERNDEVICE_ENABLED AP_WINDVANE_ENABLED
+#endif
+
+#ifndef AP_WINDVANE_BACKEND_ENABLED
+#define AP_WINDVANE_BACKEND_ENABLED AP_WINDVANE_ENABLED
```

Comment:
AP_WINDVANE_BACKEND_ENABLED should be move up and all the other libraries should check if backend is enabled, not the frontend library. See [AP_Baro_config.h](https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Baro/AP_Baro_config.h) as an example.